### DISCUSSION TITLE
Documenting public api swagger definition with source data reference links

### DISCUSSION THREAD

**Discussion by ryukyi** [amberelectric/public-api#199] [General]
Where does spot price forecast come from? I noticed in swagger definition it quotes NEM but I couldn't find specifically which route. Do spot price forecasts come from one source below? Or a combination of them?

* https://nemweb.com.au/Reports/Current/PredispatchIS_Reports/
* https://nemweb.com.au/Reports/Current/Predispatch_Sensitivities/
* https://nemweb.com.au/Reports/Current/Predispatch_Reports/
* https://nemweb.com.au/Reports/Current/Predispatch_IRSR/

Would it be possible to update the swagger definition to declare respective data sources? I can appreciate for some endpoints this may be complicated.

**Comment by madpilot** [comment]
We have a direct market connection. We only use the NEMweb website as a backup source, so this isn't really possible.
